spec/versions/changelog_version_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "Changelog version" do
+  def find_in_changelog(string)
+    File.open(changelog).each_line do |line|
+      return true if line.include? string
+    end
+
+    false
+  end
+
+  let(:changelog) { Pathname.new(__dir__).join("..", "..", "CHANGELOG.md") }
```

Comment:
What do you think about using `Rails.root` here? That way, this spec is independent of its own location in the file system.

Reply:
Since this is a gem and not a Rails application, there is no `Rails.root`. 😉

Reply:
Huh, looks like there is no way to get the root of an application in pure Ruby.